integration-cli/docker_cli_run_test.go

```diff
@@ -3363,7 +3363,7 @@ func TestRunRestartMaxRetries(t *testing.T) {
 		t.Fatal(string(out), err)
 	}
 	id := strings.TrimSpace(string(out))
-	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 5); err != nil {
+	if err := waitInspect(id, "{{ .State.Restarting }} {{ .State.Running }}", "false false", 10); err != nil {
```

Comment:
Yeah it currently runs ok even with 5 seconds, but was under heavy load (make -j14) and I suppose it was timing out for this